### PR TITLE
tictactoe game

### PR DESCRIPTION
**Pull request by chuck032**
completed majority of the game, although i was unable to make the winner visible and check if either X or O had won

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+//
```
Comment (JSChilds) [inline script.js]: Don't leave commented-out code in your final product.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+// //   }
+// // }
+// //
+// // game();
+//
+
+
+
+$(document).ready(function() {
+
```
Comment (JSChilds) [inline script.js]: All this should be indented within the function.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+var sq2 = $('data-num' | '1');
+var sq3 = $('data-num' | '2');
+var sq4 = $('data-num' | '3');
+var sq5 = $('data-num' | '4');
+var sq6 = $('data-num' | '5');
+var sq7 = $('data-num' | '6');
+var sq8 = $('data-num' | '7');
+var sq9 = $('data-num' | '8');
+
+
```
Comment (JSChilds) [inline script.js]: Don't have big, inconsistent spaces like this.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+var checkPlay = false;
+
+
+
+
+function validatePlay(squareplayed) {
+	if ( $(squareplayed).hasClass('empty') ) {
+		checkPlay = true;
+    checkWin()
```
Comment (JSChilds) [inline script.js]: Remember your semicolons.

File: script.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+var sq1 = $('data-num' | '0');
+var sq2 = $('data-num' | '1');
+var sq3 = $('data-num' | '2');
+var sq4 = $('data-num' | '3');
+var sq5 = $('data-num' | '4');
+var sq6 = $('data-num' | '5');
+var sq7 = $('data-num' | '6');
+var sq8 = $('data-num' | '7');
+var sq9 = $('data-num' | '8');
+
```
Comment (JSChilds) [inline script.js]: You aren't actually getting the elements you want back, here.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Good effort, Chuck and the logic and win conditions all look sound - you just aren't getting the right elements you need to check back from your queries. In the future, make sure when you grab an element from the DOM, you test straight away that you've got back what you expected before you start trying to do stuff to it.

**Comment by chuck032** [comment]
Was able to make the win conditions work and i've also cleaned it up so there is commented out code and indentation looks better

**Pull request merged**